heed/src/env.rs

```diff
@@ -277,9 +288,11 @@ impl EnvOpenOptions {
 }
 
 /// Returns a struct that allows to wait for the effective closing of an environment.
-pub fn env_closing_event<P: AsRef<Path>>(path: P) -> Option<EnvClosingEvent> {
+pub fn env_closing_event<P: AsRef<Path>>(path: P) -> Result<Option<EnvClosingEvent>> {
```

Comment:
It involve  public api changes here.

Reply:
Thank you, I added the `breaking` label and will create a release with the info.